script/Arcueid/Day 6/06_00_ARC06_2_2_123.txt

```diff
@@ -43,6 +43,7 @@ Professor Arach lets loose a hearty cackle. It appears she's thoroughly enjoying
 [sha:2fd169f28eec54c4eeb72426bb5b937366a7474e]{
 -- Page 252, Offset 14520.
 -- 吸血鬼という本物を知った以上、その単語を人間に使うのは許容できない。
+// to be rechecked
```

Comment:
Complaint was that this doesn't explicitly say that he's talking about real monsters existing, and those monsters are vampires. I think it's pretty implicit given the ctx, but if forced maybe
> Now that I know about the existence of vampires, the genuine article, I can't abide...
> Now that I know monsters - vampires - are real, I can't abide.

actually heck, maybe ruby it
> Now that I know <monsters|vampires> - are real, I can't abide.

Reply:
I'm kinda indifferent to changing this line, this definitely strikes me as a stylistic thing rather than a meaning thing. That said, if we were to take one, I'd go with a light modification on your suggestion:
> Now that I know about the existence of vampires--the real thing--I can't abide...